mscore/menus.cpp

```diff
@@ -1569,8 +1569,10 @@ void MuseScore::setAdvancedPalette()
       preferences.paletteCellList.clear();
       for (int i = 0; i < shortcuts.size(); i++) {
             PaletteCell* cell = cells[i];
-            if (cell->name == "Show More")
+            if (cell->name == "Show More") {
```

Comment:
I wonder if it is also worth making this a boolean flag in the cell, set when we create it, rather than keying off the name?  Or if perhaps something like checking if cell->element == 0?